manifest.yaml

```diff
@@ -0,0 +1,29 @@
+AzureCore:
```

Comment:
What is this file used for? Ideally we should try to avoid yet another list of projects and versions to maintain if we can.

Reply:
Right now the script only uses the source and mirror entries. Unless we have a deterministic way of mapping SDKs to repos, I don't see how we will avoid this. `Package.swift` files don't actually specify versions, so the idea behind version would be a simple place to acquire the correct version for tagging. We could potentially pull that from the podspec though.

Reply:
Ideally we would only have one place that defines the version. If swift doesn't need it perhaps we could use the pod. I know some languages also embed the version in source code so it can also be used by code if needed.

Reply:
As for the other metadata here could we gather it from the swift files instead of explicitly listing it? For most languages we essentially traverse the sdk looking for some sort of project file and then read the metadata from there.